Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10-slim
+
+RUN apt update -y && apt install -y curl git python3-pip && apt clean && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /goodbot_api
+
+COPY requirements.txt .
+
+RUN python3 -m pip install --upgrade pip
+
+RUN python3 -m pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+EXPOSE 8000
+
+CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "8000", "--reload"]
```

Comment:
We're running a new version of fastapi so I think we need to change this to:
```
CMD ["fastapi", "run", "app/api/main.py", "--host", "0.0.0.0", "--port", "80"]
```

Reply:
I did some research and tried it before. It doesn't work as we want it. So the best option was using uvicorn. Still I will check it again to see if it is functioning using fastapi or not!